lib/rubocop/cop/lint/number_conversion.rb

```diff
@@ -53,6 +53,15 @@ def on_send(node)
           end
         end
 
+        def autocorrect(node)
+          lambda do |corrector|
+            to_method(node) do |receiver, _to_method|
+              corrector.replace(node.loc.expression,
+                                correct_method(node, receiver))
```

Comment:
you can just use `node.receiver` and skip the `to_method` matching, but I'm fine with your approach as well